synapse/lib/storm.py

```diff
@@ -2416,7 +2413,8 @@ def getDescr(self):
         return self.__class__.__doc__
 
     def getArgParser(self):
-        return Parser(prog=self.getName(), descr=self.getDescr(), model=self.runt.model)
+        cdef = getattr(self, 'cdef', None)
```

Comment:
This seems like it should be on PureCmd since those are the only Cmds that have a cdef